packages/react-core/src/demos/DescriptionList/examples/DescriptionListDrawer.tsx

```diff
@@ -0,0 +1,167 @@
+            <DescriptionListDescription>Nod selector is not available at this time</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Tolerations</DescriptionListTerm>
+            <DescriptionListDescription>No tolerations</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Annotations</DescriptionListTerm>
+            <DescriptionListDescription>No annotaions</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Status</DescriptionListTerm>
+            <DescriptionListDescription>Active</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Created at:</DescriptionListTerm>
+            <DescriptionListDescription>3 minutes agot</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Pod selector</DescriptionListTerm>
+            <DescriptionListDescription>
+              <Button variant="link" isInline icon={<PlusCircleIcon />}>
+                app=MyApp
+              </Button>
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Session affinity</DescriptionListTerm>
+            <DescriptionListDescription>None</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Latest version</DescriptionListTerm>
+            <DescriptionListDescription>1.0</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Update strategy</DescriptionListTerm>
+            <DescriptionListDescription>Rolling</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Timeout</DescriptionListTerm>
+            <DescriptionListDescription>600 seconds</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Max available</DescriptionListTerm>
+            <DescriptionListDescription>25% of 1 pod</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Max surge</DescriptionListTerm>
+            <DescriptionListDescription>25% greater than 1 pod</DescriptionListDescription>
+          </DescriptionListGroup>
+        </DescriptionList>
+      </DrawerPanelBody>
+    </DrawerPanelContent>
+  );
+
+  const drawerContent = (
+    <Gallery hasGutter>
+      <GalleryItem key={0}>
+        <Card>
+          <CardBody>
+            <Button variant="link" onClick={onOpenDrawer} tabIndex={isExpanded ? 0 : -1} ref={drawerRef}>
```

Comment:
If you add `isInline`, it will remove the padding around the button and the card with the button will look like the other cards

```suggestion
            <Button variant="link" isInline onClick={onOpenDrawer} tabIndex={isExpanded ? 0 : -1} ref={drawerRef}>
```

<img width="1820" alt="Screenshot 2024-04-23 at 6 42 02 PM" src="https://github.com/patternfly/patternfly-react/assets/35148959/7d380a72-124b-4e83-8ac5-9dcb08649537">